Documentation/ceph-osd-mgmt.md

```diff
@@ -63,23 +60,51 @@ through the removal process:
 If all the PGs are `active+clean` and there are no warnings about being low on space, this means the data is fully replicated
 and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly clean and you will need to proceed anyway.
 
-### From the Toolbox
+### Prevent the rook operator to re-create the OSD pod
```

Comment:
I don't understand why this title is needed. How about removing it, and just keep the host-based and pvc-based titles as third-level?
```suggestion

```

Reply:
Indeed. Removed.